WORKSPACE

```diff
@@ -382,11 +382,14 @@ http_archive(
 http_archive(
     name = "aws-sdk-cpp",
     build_file = "//third_party:aws-sdk-cpp.BUILD",
-    sha256 = "8724a2c3f00478e5f9af00d1d9bc67b7a0a557cc587a10f7097b1257008c2e68",
-    strip_prefix = "aws-sdk-cpp-1.7.270",
+    patch_cmds = [
+        """sed -i.bak 's/UUID::RandomUUID/Aws::Utils::UUID::RandomUUID/g' aws-cpp-sdk-core/source/client/AWSClient.cpp""",
```

Comment:
@yongtang any reason for this replacement?

Reply:
@kvignesh1420 On Windows UUID is a defined symbol so a build failure will occur:
```
external/aws-sdk-cpp/aws-cpp-sdk-core/source/client/AWSClient.cpp(215): error C2872: 'UUID': ambiguous symbol
```

As such we have to declare `Aws::Utils::UUID::RandomUUID` in full namespace (not starting with `UUID::RandomUUID`)

Reply:
Ok